django/contrib/postgres/indexes.py

```diff
@@ -11,11 +11,11 @@ class BrinIndex(Index):
     # applicable.
     max_name_length = 31
 
-    def __init__(self, fields=[], name=None, pages_per_range=None):
+    def __init__(self, fields=[], name=None, pages_per_range=None, *, db_tablespace=None):
```

Comment:
I prepared separate PR for that #8665. I wasn't sure if I should leave `fields=[], name=None` as positional args.